tsa-test/src/test/resources/checkers/bounce.fc

```diff
@@ -0,0 +1,31 @@
+#include "../imports/stdlib.fc";
+#include "../imports/tsa_functions.fc";
+
+() on_internal_message_send(int balance, int msg_value, cell in_msg_full, slice msg_body, int input_id) impure method_id {
+    
+    slice cs = in_msg_full.begin_parse();
+    int flags = cs~load_uint(4);
+    tsa_assert_not(flags & 1);
+
+    slice body_copy = msg_body;
+    int op = body_copy~load_uint(32);
```

Comment:
You don't need to create `body_copy` in this case, since slices are immutable in TVM